mmv1/third_party/terraform/transport/error_retry_predicates.go

```diff
@@ -115,6 +115,21 @@ func is409OperationInProgressError(err error) (bool, string) {
 	return false, ""
 }
 
+// Code Repository Index is a long running operation
+// The resource takes time to change it's state from "CREATING" to "ACTIVE"
+func IsCodeRepositoryIndexUnreadyError(err error) (bool, string) {
+	gerr, ok := err.(*googleapi.Error)
+	if !ok {
+		return false, ""
+	}
+
+	if gerr.Code == 409 && strings.Contains(gerr.Body, "parent resource not in ready state") {
```

Comment:
FYI when we start depending on error message contents it makes it extremely difficult to change the error messages on the API side in the future. That's the state of the world today so it's not unreasonable to add this, but it does have downsides